stablehlo/transforms/VhloLegalizeToStablehlo.cpp

```diff
@@ -745,10 +745,11 @@ LogicalResult removeDefaults(const OpConversionPattern<VhloOpTy>& pattern,
       eraseAttrs(vhloAttrs, "api_version");
     if (isEmptyArray(vhloOp.getCalledComputations()))
       eraseAttrs(vhloAttrs, "called_computations");
-    if (isEmptyArray(vhloOp.getOperandLayouts()))
+    if (isEmptyArray(vhloOp.getOperandLayouts()) &&
```

Comment:
Does the problematic case has only one of operand and result layouts empty?

Reply:
Our verification currently says "must specify both operand+result layout, or neither".

So if there's empty operands or results, we end up stripping the empty one, leading to verification failures